packages/frontend/src/pages/positions/pending/offers.tsx

```diff
@@ -0,0 +1,66 @@
+import { AssetId, Timestamp } from '@explorer/types'
+import React from 'react'
+
+import { AssetCell } from '../../common/AssetCell'
+import { formatCurrencyUnits, formatRelativeTime } from '../../formatting'
+import { OfferType } from '../../offers'
+import { PendingRow } from './row'
+
+interface OfferEntry {
+  type: OfferType
+  assetId: AssetId
+  amountSynthetic: bigint
+  amountCollateral: bigint
+}
+
+interface CreatedOfferEntry extends OfferEntry {
+  status: 'created'
+}
+
+interface MatchedOfferEntry extends OfferEntry {
+  status: 'matched'
+  expirationTime: Timestamp
+}
+
+export type PendingOfferEntry = CreatedOfferEntry | MatchedOfferEntry
+
+export function PendingOffers({
+  offers,
+}: {
+  readonly offers: readonly PendingOfferEntry[]
+}) {
```

Comment:
```suggestion
interface PendingOffersProps {
  readonly offers: readonly PendingOfferEntry[]
}

export function PendingOffers({ offers }: PendingOffersProps) {
```

Makes it more readable